packages/protocol/runTests.js

```diff
@@ -3,21 +3,23 @@ const glob = require('glob-fs')({
   gitignore: false,
 })
 const { exec } = require('./lib/test-utils')
-const network = require('./truffle.js').networks.development
 const minimist = require('minimist')
 
 const sleep = (seconds) => new Promise((resolve) => setTimeout(resolve, 1000 * seconds))
 
+// As documented https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+const isCI = process.env.CI === 'true'
+
 async function startGanache() {
   const server = ganache.server({
-    default_balance_ether: network.defaultBalance,
-    network_id: network.network_id,
-    mnemonic: network.mnemonic,
-    gasPrice: network.gasPrice,
+    default_balance_ether: 1000000,
+    gasPrice: 0,
```

Comment:
This was originally inside truffle network config. I've just moved it.

Why 0? My guess, it makes simpler to make asserts on balances; since you don't need to take into account the amount spent in gas.